facebook_scraper.py

```diff
@@ -259,3 +260,23 @@ def _login_user(email, password):
     _session.post(_base_url + login_action, data={'email': email, 'pass': password})
     if 'c_user' not in _session.cookies:
         warnings.warn('login unsuccessful')
+
+def get_posts_as_csv(account, pages=10, timeout=5, sleep=0, credentials=None):
+    """
```

Comment:
The name `get_posts_as_csv` is confusing since it writes the output to a file, while `get_posts` writes the output to stdio